runtime/service.go

```diff
@@ -52,6 +53,7 @@ import (
 const (
 	defaultVsockPort     = 10789
 	supportedMountFSType = "ext4"
+	minFCDriveImageSize  = 512
```

Comment:
Bytes?

Reply:
Yea, will change to `minFCDriveImageSizeBytes`